Library/templates/admin_layout.html

```diff
@@ -33,18 +35,23 @@
                     <li class='nav-item active ml-4'>
                          <a class='nav-link' href='/remove/book'>Remove Book</a>
                     </li>
+                    <li class='nav-item active ml-4'>
+                         <a class='nav-link' href='/show/students'>Show Students</a>
+                    </li>
                     <li class='nav-item active ml-4'>
                          <a class='nav-link' href='/admin/logout'>Logout</a>
                     </li>
                </ul>
           </div>
      </nav>
-     <div class="container" style="color: white;text-shadow: 2px 2px 12px #000000;">
+     <div class="container" style="color: grey;">
           {% block content %}{% endblock %}
-          <hr style="border: 5px solid white;border-radius: 5px;" />
-          <footer>
-               <p>&copy; {{ year }} - Library Management System - Kumar Aditya</p>
-          </footer>
+          <div valign="bottom">
+               <hr style="border: 5px solid grey;border-radius: 5px;" />
+               <footer>
+                    <p>&copy; {{ year }} - Library Management System - Harshit Soora</p>

```

Comment:
You can add your name on your repo, let it be intact for this repo

Reply:
Ohh sorry I should have removed that before committing